src/backend/pipeline/cont_scheduler.c

```diff
@@ -673,6 +673,10 @@ ContQuerySchedulerMain(int argc, char *argv[])
 	ContQuerySchedulerShmem->scheduler_pid = MyProcPid;
 
 	dbs = get_database_list();
+	if (list_length(dbs) * TOTAL_SLOTS > max_worker_processes)
```

Comment:
We should probably throw an error and not start any continuous query procs. Seeing the state where one db gets all the workers and the other one doesn't seems like something we shouldn't do.